packages/react-components/src/components/InputBoxComponent.tsx

```diff
@@ -992,7 +1015,7 @@ const handleMentionTagUpdate = (props: MentionTagUpdateProps): MentionTagUpdateR
 
 /* @conditional-compile-remove(mention) */
 /**
- * Get close tag information if exists otherwise return information as for self closing tag
+ * Get closing tag information
```

Comment:
This should be `Type of a Closing Tag Info Result`?
Also, should we change the name of ClosingTagInfoResult into either ClosingTagInfo or ClosingTagResult?
I prefer ClosingTagInfo since this type doesn't seem to be a result of anything.

Reply:
Updated the description a little bit to be clearer as discussed